### PR TITLE
Switchy Proxy Support

### LINKED ISSUE #5: [Request] Switchy Proxy Support
We threw this around a couple of times during the mad 2.0 rush but we should file it properly now that's over ^^

[Switchy Proxy](https://modrinth.com/mod/switchy-proxy) provides similar functionality to pk and tb's message proxying.
 
It adds a module (ProxyModule) than can have an arbitrary number of "tags" (ProxyTag).These can be constructed using a prefix and a suffix, or parsed from a string containing one instance of the substring `text`.

These should be mutually intelligible with proxy tags from pk and tb, and it would be very useful to be able to import them directly!

### PR DESCRIPTION
**Pull request by sleepingdragoninn**
closes #5 

When [Switchy Proxy](https://modrinth.com/mod/switchy-proxy) is installed, importing from PK from token or json will populate presets with `proxies` modules, allowing for proxied messages.

### CODE REVIEW

File: src/main/kotlin/platform/TUL.kt [hunk 1]
```diff
@@ -28,7 +28,8 @@ object TUL {
                                 tupper.name,
                                 tupper.nick,
                                 null,
-                                null
+                                null,
+                                arrayListOf()
```
Comment (Oliver-makes-code) [inline src/main/kotlin/platform/TUL.kt]: Tupperbox members have a `brackets` field, which is a 2 element array containing a prefix and a suffix

**Review by Oliver-makes-code — CHANGES_REQUESTED** [review]
Could this be implemented?
Here's an example [tupper export](https://github.com/The-ProxyFox-Group/ProxyFox/blob/master/modules/database/src/test/resources/dev/proxyfox/database/systems/pass/Tupperbox.json)

File: src/main/kotlin/platform/TUL.kt (hunk 1 above)
Comment (sleepingdragoninn) [reply src/main/kotlin/platform/TUL.kt]: will do!

Comment (sleepingdragoninn) [reply src/main/kotlin/platform/TUL.kt]: Added support and confirmed working! thanks for the spec info.

javaw_RjHuU64yX8

**Review by Oliver-makes-code — APPROVED** [review]

**Pull request merged**